tests/ha/rear_backup.pm

```diff
@@ -47,6 +47,9 @@ sub run {
         my $local_conf = '/etc/rear/local.conf';
         assert_script_run("curl -f -v " . data_url('ha/rear_local.conf') . " -o $local_conf");
         file_content_replace("$local_conf", q(%BACKUP_URL%) => $backup_url);
+        # add hostname and archtecture as workaround for TEAM-9212
+        assert_script_run("echo 'OUTPUT_PREFIX=\"\$HOSTNAME-\$REAL_MACHINE\"' >> $local_conf");
+        assert_script_run("echo 'NETFS_PREFIX=\"\$OUTPUT_PREFIX\"' >> $local_conf");
```

Comment:
Suggestion: better add these 2 lines to the configuration file in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/ha/rear_local.conf

That way you avoid sending 2 additional commands to the SUT.

Since the file is only used by this module, it's safe to apply the changes in the configuration file directly.

Reply:
Updated VRs and added changes on local.conf.